src/main/scala/mesosphere/marathon/core/matcher/manager/OfferMatcherManagerModule.scala

```diff
@@ -41,6 +41,6 @@ class OfferMatcherManagerModule(
     * offers.
     */
   val globalOfferMatcherWantsOffers: Observable[Boolean] = offersWanted
-  val globalOfferMatcher: OfferMatcher = new ActorOfferMatcher(offerMatcherMultiplexer, None, homeRegion)
+  val globalOfferMatcher: OfferMatcher = new ActorOfferMatcher(offerMatcherMultiplexer, None)
```

Comment:
Why won't we pass the region anymore? Is this a bug?

Reply:
I'm probably missing the obvious but it seems to me we are not matching the region anymore. Where is that done?

Reply:
In the resource matcher